src/Chapter3.hs

```diff
@@ -907,9 +1023,34 @@ Implement instances of "Append" for the following types:
   ✧ *(Challenge): "Maybe" where append is appending of values inside "Just" constructors
 
 -}
+newtype Gold = Gold { goldValue :: Int }
 class Append a where
-    append :: a -> a -> a
+	append :: a -> a -> a
+
+instance Append Gold where
+	append :: Gold -> Gold -> Gold
+	append x1 x2 = Gold ((goldValue x1) + (goldValue x2))
```

Comment:
Your solution is correct :+1: Alternatively, you can pattern match on the `Gold` constructor directly:
```suggestion
	append (Gold x1) (Gold x2) = Gold (x1 + x2)
```